Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/git_unittest.py

```diff
@@ -599,6 +599,19 @@ def test_merge_base(self):
                 '2@main',
             )
 
+    def test_is_pull_request_branch(self):
+        with mocks.local.Git(self.path, remotes={
+            'origin': 'git@github.example.com:WebKit/WebKit.git'
+        }):
+            self.assertEqual(local.Git(self.path).is_pull_request_branch('eng/12345', source_remote=local.Git(self.path).source_remotes()[0]), True)
```

Comment:
Can you avoid so much of the duplication here, `repo = local.Git(self.path)` and `source_remote = local.Git(self.path).source_remotes()[0]`, make the lines shorter and easier to read?